src/app/dashboard/home/home.component.ts

```diff
@@ -10,16 +10,16 @@ import { UserService } from 'src/app/services/user.service';
 })
 export class HomeComponent implements OnInit, OnDestroy {
   users: User[] = [];
-  usersSubscription: Subscription = new Subscription();
+  private destroy$ = new Subject();
 
   constructor(private userService: UserService) {}
   ngOnInit() {
-    this.usersSubscription = this.userService.usersObservable.subscribe(
-      (users) => (this.users = users)
-    );
+    this.userService.usersObservable
+      .pipe(takeUntil(this.destroy$))
+      .subscribe((users) => (this.users = users));
   }
 
   ngOnDestroy() {
-    this.usersSubscription.unsubscribe();
+    this.destroy$.complete();
```

Comment:
The `destroy` subject needs to emit a value before completing so that all the subscriptions can unsubscribe.